content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -45,6 +45,8 @@ In this case, the virtual services will route all traffic to `v1` of each micros
     Because configuration propagation is eventually consistent, wait a few seconds
     for the virtual services to take effect.
 
+    {{< idea_icon >}} If you haven't already applied destination rules, follow follow the instructions in [Apply Default Destination Rules](/docs/examples/bookinfo/#apply-default-destination-rules).
```

Comment:
follow used twice.  I think `>` would be preferred to an `{{< idea_icon >}}`.  Generally we escalate in the docs bold, and then with `>` and finally emojis `{{< idea icon >}}`.

Reply:
@irisdingbj can you please remove the extra "follow"